src/OrleansProviders/ConfigExtensions.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Reflection;
+
+
+namespace Orleans.Runtime.Configuration
+{
+    /// <summary>
+    /// Extension methods for configuration classes specific to OrleansProviders.dll 
+    /// </summary>
+    public static class CongigExtensions
+    {
+        /// <summary>
+        /// Adds default storage provider named "MemoryStore" of type Orleans.Storage.MemoryStorage
+        /// </summary>
+        /// <param name="config">Cluster configuration object to add provider to</param>
+        /// <returns></returns>
+        public static ClusterConfiguration AddDefaultMemoryStore(this ClusterConfiguration config)
+        {
+            config.Globals.RegisterStorageProvider<Storage.MemoryStorage>("MemoryStore");
+            return config;
```

Comment:
Are we sure you want to start creating a "fluent API" before designing exactly how it will look like?
Wouldn't it be safer to just return void here for now, and potentially make it fluent after we re-design the config? I'm fine either way, it's just that `void` is more conservative.